dev-docs/show-prebid-ads-on-amp-pages.md

```diff
@@ -333,3 +333,4 @@ To review that Prebid on AMP is working properly the following aspects can be lo
 
 [PBS]: /prebid-server/overview/prebid-server-overview.html
 [callout-vendors.js]: https://github.com/ampproject/amphtml/blob/master/src/service/real-time-config/callout-vendors.js
+[RTC-Overview]: https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/rtc-documentation.md
```

Comment:
I thought this would fix the broken link on the top of the page, but it doesn't. Why?

Reply:
NVM. The fix works. Searching on the preview page redirects you back to the main page. Doh!